crates/nu-protocol/src/value/filesize.rs

```diff
@@ -1,13 +1,442 @@
-use crate::Config;
+use crate::{Config, FromValue, IntoValue, ShellError, Span, Type, Value};
 use byte_unit::UnitType;
```

Comment:
I am wondering if with all the code you have written, we can basically get rid of the `byte_unit` dependency? Last time I checked we only used a few parts with many of our own types or formatting on top.

Reply:
Yep, in my other branch, I was able to get rid of the dependency.